index.bs

```diff
@@ -1963,31 +1970,18 @@ partial interface mixin WebGLRenderingContextBase {
 };
 </pre>
 
-When a user agent implements this specification it MUST set a <dfn>XR compatible</dfn> boolean, initially set to <code>false</code>, on every {{WebGLRenderingContextBase}}. Once the [=XR compatible=] boolean is set to <code>true</code>, the context can be used with layers for any {{XRSession}} requested from the current [=XRSystem/immersive XR device=].
+When a user agent implements this specification it MUST set a <dfn>XR compatible</dfn> boolean, initially set to <code>false</code>, on every {{WebGLRenderingContextBase}}. Once the [=XR compatible=] boolean is set to <code>true</code>, the context can be used with layers for any {{XRSession}} requested from the current [=immersive XR device=].
 
-Note: This flag should only be set if you plan to start an XR session. On some systems this flag may turn on a high powered discrete GPU, for example, or proxy all commands to an on-device GPU. If you are in a situation where you may switch to XR later, consider calling {{WebGLRenderingContextBase/makeXRCompatible()}} when that decision is made.
+Note: This flag introduces slow synchronous behavior and is deprecated. Consider calling {{WebGLRenderingContextBase/makeXRCompatible()}} instead for an asynchronous solution.
```

Comment:
I don't know that we want to call it "deprecated" just yet. That'll definitely be the case if/when we get a `getContextAsync` method. Sync-shaming the method seems appropriate, though.